### PR TITLE
Feature/NT-166 hourglass UI creation

### PR DESCRIPTION
**Pull request by jortdr**
Hourglass UI added
Coded falling sand
Cleaned up existing code.

### CODE REVIEW

File: Assets/Scripts/Presenters/HourglassPresenter.cs [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+namespace Presenters
+{
+    public class HourglassPresenter : MonoBehaviour
+    {
+        private float _lerpPercentage;
+
+        #region Sand
+
+        /* Top Sand */
+        public Transform sandTop;
```
Comment (PatrickSlag) [inline Assets/Scripts/Presenters/HourglassPresenter.cs]: i think you should use [SerializeField] private to get unity object instead of just public

**Review by PatrickSlag — CHANGES_REQUESTED** [review]
can there be a test scene or gif of how this works/looks?

**Comment by jortdr** [comment]
Untitled-2-ezgif com-optimize

@PatrickSlag here's a gif of how it works

**Review by PatrickSlag — DISMISSED** [review]

File: .vsconfig [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
```
Comment (VarioCode) [inline .vsconfig:1]: Shouldn't this be in the git ignore?

**Review by VarioCode — CHANGES_REQUESTED** [review]

File: .vsconfig (hunk 2 above)
Comment (VarioCode) [inline .vsconfig]: Shouldn't this be in the git ignore?

**Review by VarioCode — APPROVED** [review]

**Review by PatrickSlag — APPROVED** [review]

**Pull request merged**